src/fmu/dataio/export/rms/inplace_volumes.py

```diff
@@ -57,15 +60,22 @@
 }
 
 
-@dataclass
-class _ExportVolumetricsRMS:
-    project: Any
-    grid_name: str
-    volume_job_name: str
+class _ExportVolumetricsRMS(SimpleExportRMSBase):
```

Comment:
Converted this to a regular class while here, to align it with the other Simple Exporters